apps/omg/lib/omg/output.ex

```diff
@@ -30,64 +31,44 @@ defmodule OMG.Output do
 
   defstruct [:output_type, :owner, :currency, :amount]
 
-  alias OMG.RawData
-  alias OMG.Utxo
-
-  require Utxo
-
-  @output_types_modules OMG.WireFormatTypes.output_type_modules()
-  @output_types Map.keys(@output_types_modules)
-
-  def reconstruct([raw_type, rlp_decoded_chunks]) when is_binary(raw_type) do
-    case RawData.parse_uint256(raw_type) do
-      {:ok, output_type} when output_type in @output_types ->
-        reconstruct([output_type, rlp_decoded_chunks])
-
-      {:ok, _unrecognized_type} ->
-        {:error, :unrecognized_output_type}
-    end
-  end
-
   @doc """
   Reconstructs the structure from a list of RLP items
   """
```

Comment:
are there no dialyzer specs available?

Reply:
@InoMurko no there weren't dialyzer's availlable. In the follow up PR to this, I'll add them to `OMG.Utxo`. The end goal is to consolidate this back into that module.

Reply:
ok